ui/perfherder/SelectorCard.jsx

```diff
@@ -76,8 +99,11 @@ export default class SelectorCard extends React.Component {
     });
   };
 
-  updateData = selectedRepo => {
+  updateRevisions = selectedRepo => {
     const { updateState, projectState } = this.props;
+    if (this.state.invalidProject) {
+      this.setState({ invalidProject: false });
+    }
```

Comment:
I should add a comment in the code here. The reason why `invalidProject` is reset is in case a user types an invalid repo as a queryParam, they see a message telling them so, and if they use the project dropdown to select a new one (rather than updating it via the queryparam), we need to do away with the error message.